packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporterBrowserBase.ts

```diff
@@ -94,7 +101,7 @@ export abstract class CollectorExporterBrowserBase<
       if (this._useXHR) {
         sendWithXhr(body, this.url, this._headers, _onSuccess, _onError);
       } else {
-        sendWithBeacon(body, this.url, _onSuccess, _onError);
+        sendWithBeacon(body, this.url, this._blobPropertBag, _onSuccess, _onError);
```

Comment:
My original comment was that as the ```sendWithBeacon``` was public we should support passing a contentType (which you have and I don't have an issue with that part).

The issue here is that the ```send``` function is explicitly creating a JSON payload and therefore this function is the one that should be setting the contentType (which it only does indirectly because of the default).

Suggestions: 
- Either remove the ```_blobPropertBag``` completly and just create a blobPropertyBag here when calling the ```sendWithBeacon``` (Preferred as there is currently only 2 options for the blobProperty options anyway)
- Create a local blobPropertyBag instance (using the initialized version) as a base, but replace the contentType (as this is sending Json) (as above as there is only 2 options for the blob options this is overkill as well)